tests/unit/spec/base/mozilla-traffic-cop.js

```diff
@@ -0,0 +1,191 @@
+/* For reference read the Jasmine and Sinon docs
+ * Jasmine docs: http://pivotal.github.io/jasmine/
+ * Sinon docs: http://sinonjs.org/docs/
+ */
+
+/* global describe, beforeEach, afterEach, it, expect */
+
+describe('mozilla-traffic-cop.js', function () {
+    'use strict';
+
+    beforeEach(function() {
+        // stub out Mozilla.Cookie lib
+        Mozilla.Cookies = {
```

Comment:
Instead of creating a dummy object which still hangs around after the test finishes, you can use `sinon` to create temporary stubs for each function i.e.

```
window.Mozilla.Cookies = sinon.stub();
window.Mozilla.Cookies.setItem = sinon.stub();
window.Mozilla.Cookies.getItem = sinon.stub();
window.Mozilla.Cookies.hasItem = sinon.stub();
```